pkg/networkmanager/firewall.jsx

```diff
@@ -701,26 +702,27 @@ class ActivateZoneModal extends React.Component {
                    </>}
             >
                 <Form isHorizontal>
-                    <FormGroup label={ _("Trust level") }>
-                        <div role="group" className="add-zone-zones">
-                            <div className="add-zone-zones-firewalld">
-                                <legend>{ _("Sorted from least trusted to most trusted") }</legend>
+                    <FormGroup label={ _("Trust level") } className="add-zone-zones">
+                        <Flex>
+                            <FlexItem className="add-zone-zones-firewalld">
+                                <legend>{ _("Sorted from least to most trusted") }</legend>
                                 { zones.filter(z => firewall.predefinedZones.indexOf(z) !== -1).sort((a, b) => firewall.predefinedZones.indexOf(a) - firewall.predefinedZones.indexOf(b))
                                         .map(z =>
                                             <Radio key={z} id={z} name="zone" value={z}
-                                                   onChange={(value, e) => this.onChange("zone", e.target.value)}
-                                                   label={ firewall.zones[z].id } />
+                                                    defaultChecked={ firewall.predefinedZones.indexOf(z) === 1 }
+                                                    onChange={(value, e) => this.onChange("zone", e.target.value)}
+                                                    label={ firewall.zones[z].id } />
                                         )}
-                            </div>
-                            <div className="add-zone-zones-custom">
+                            </FlexItem>
+                            <FlexItem className="add-zone-zones-custom">
                                 { customZones.length > 0 && <legend>{ _("Custom zones") }</legend> }
                                 { customZones.map(z =>
                                     <Radio key={z} id={z} name="zone" value={z}
-                                           onChange={(value, e) => this.onChange("zone", e.target.value)}
-                                           label={ firewall.zones[z].id } />
+                                            onChange={(value, e) => this.onChange("zone", e.target.value)}
+                                            label={ firewall.zones[z].id } />
```

Comment:
Oh weird; did I mess up the indentation? If so, whoops. ESLint didn't complain though.

Reply:
I fixed it back manually. You probably have some auto indent feature in your editor?

Reply:
Yeah, I was indenting / re-indenting a few blocks, and I guess I — or my editor — got it wrong.